lib/Partitioner/Partitioner.cpp

```diff
@@ -425,6 +426,39 @@ Expected<DAGListTy> Partitioner::createDAGWithoutPartition(
   }
 
   NodeToFunctionMap mapping;
+  for (auto func : module_->getFunctions()) {
+    mapping.createPartition(func, backendName);
+
+    GraphMemInfo graphMem;
+    graphMem.constMemSize = func->getParent()->getConstantsSize();
+
+    // Walk thru all the nodes to compute input and output mem size.
+    for (auto &node : func->getNodes()) {
```

Comment:
For calculating the graphMem of a single function, I think it is better write it as a function in PartitionerUtils.cpp. Then we can write a unittest in similar to https://github.com/pytorch/glow/blob/master/tests/unittests/PartitionerTest.cpp#L727
.